File: src/app/entities/rides/ride-company.ts
import { Ride } from './ride';
import { RideLocationTransfers } from '../flights/location-transfers';

export class RideCompany {
    id: number;
    name: string;
    image: string;
    address: string;
    promoDescription: string;
    rating: number;
    rides: Array<Ride>;
    numberOfTransfers:number;
    rideLocationTransfers: Array<RideLocationTransfers>;
    userID:string
    constructor()
    {
    }
}

File: projectBackend/projectBackend/Models/RideCompany.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace projectBackend.Models
{
  public class RideCompany
  {
    [Key]
    public int ID { get; set; }
    public string Name { get; set; }
    public string Image { get; set; }
    public string Address { get; set; }
    public string PromoDescription { get; set; }
    public int Rating { get; set; }
    public virtual List<Ride> Rides { get; set; }
 
  }
}
